src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,11 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
```

Comment:
You have not used the getPrintableString(Printable... printables) in an appropriate place.